tests/scripts/thread-cert/border_router/test_single_border_router.py

```diff
@@ -162,7 +162,7 @@ def test(self):
         br.remove_prefix(ON_MESH_PREFIX2)
         br.register_netdata()
 
-        self.simulator.go(10)
+        self.simulator.go(310)
```

Comment:
Same as previous comment: the netdata prefix and route related tests should be done sooner (after the original 10 sec), while only SLAAC address related tests need to be delayed by additional 300 sec.

Reply:
It is possible but then requires a good amount of change to this test to take into account the deprecating addresses. 

I would take a look at how much change is needed. If it become a re-write of the good chunk of the test itself, then I prefer to use longer wait time and keep the test unchanged.

Reply:
Ok, what I had in mind is just inserting an additional `go()` and splitting the tests in two parts. The changes that take effect fast (few seconds) are then tested first. The rest (mostly the OMR address removal) later. When doing this, it would be a better test I believe.

Reply:
After reviewing both tests, I decided to retain `go(310)` as the main step since it's immediately followed by `collect_ipaddrs()`.  Updating these older tests to handle address deprecation would be extensive. Additionally,  similar scenarios are already well-covered in newer tests. It might be preferable to eventually deprecate these older tests entirely.

Reply:
Ok, if there are other tests covering this area then we can keep the `go(310)` here.